factory-containers/docker-app-publish.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh -e
```

Comment:
$ checkbashisms factory-containers/docker-app-publish.sh
possible bashism in factory-containers/docker-app-publish.sh line 5 (should be '.', not 'source'):
source $HERE/../helpers.sh
possible bashism in factory-containers/docker-app-publish.sh line 10 (${foo:3[:1]}):
TAG=${GIT_SHA:0:6}
possible bashism in factory-containers/docker-app-publish.sh line 25 (alternative test command ([[ foo ]] should be [ foo ])):
if [[ "$sha" = "$newsha" ]] && [[ -n "$apps" ]] ; then

Same here.